### ISSUE TITLE
Add support for action buttons

### ISSUE THREAD

**Issue by sampaiodiego** [RocketChat/Rocket.Chat.Android#2036]
## Describe the bug
As I can see, action buttons are already being rendered, but they're not calling the slash command when applicable. This is important because of the current state of Rocket.Chat Apps, as they're using action buttons to interact with users, when clicking such buttons, a slash command is called.

## To Reproduce
1. Click on an action button
2. See a message being sent

## Expected behavior
A slash command should be called instead of sending a message.

Also as shown on the screenshot below, emojis and markdown are not being rendered on the attachment fields.

## Screenshots
![screenshot_20190225-092617](https://user-images.githubusercontent.com/8591547/53337404-148b1600-38e0-11e9-9631-90396ca7d1d5.png)

The second message should not be sent, it should call it as a slash command instead.

## Devices and Versions

Your Rocket.Chat.Android version: 3.2.0 build 2057
Your Rocket.Chat Server version: 0.74.3

Mobile device model and OS version: Mi A2 - Android Pie

## Additional context

Below is a compact version of a message object containing the relevant fields for action buttons:
```json
{
  "_id": "ryWoB9zYFzWEaPABs",
  "rid": "XkhRGsh7SGkDJPkbj",
  "msg": "_Is poll app working?_",
  "attachments": [
    {
      "color": "#73a7ce",
      "text": ":one: Yes `0`\n:two: No `0`\n:three: Some times `0`"
    },
    {
      "color": "#73a7ce",
      "button_alignment": "horizontal",
      "actions": [
        {
          "type": "button",
          "text": "1",
          "msg_in_chat_window": true,
          "msg": "/vote cf4169ea-cab1-46a3-b798-dfddea2db474 0"
        },
        {
          "type": "button",
          "text": "2",
          "msg_in_chat_window": true,
          "msg": "/vote cf4169ea-cab1-46a3-b798-dfddea2db474 1"
        },
        {
          "type": "button",
          "text": "3",
          "msg_in_chat_window": true,
          "msg": "/vote cf4169ea-cab1-46a3-b798-dfddea2db474 2"
        }
      ]
    }
  ]
}
```

So basically, if `msg_in_chat_window` is `true`, the `msg`  property will be the slash command executed when tapped.

If you need more information, please just let me know.

**Comment by graywolf336** [comment]
There are even more action buttons now. We need to work on documentation.

**Comment by lorek123** [comment]
Any news on this?
